InfusedFoods/ja_jp.json

```diff
@@ -1,3 +1,3 @@
 {
-    "block.infusedfoods.infusion_station": "輸液ステーション"
-}
\ No newline at end of file
+    "block.infusedfoods.infusion_station": "注入台"
```

Comment:
The original value of this translates to `Infusion Station` which is the original name for the block.
Your change / correction translates to `Injection Table`, I prefer the original as it sticks to what the US name of the block is.